spec/models/heart_rate_data_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe HeartRateData, type: :model do
+
+  let(:user) { FactoryGirl.create(:user) }
```

Comment:
Rather than `create`, I think you can `build_stubbed` here and avoid a round trip to the database.